app/views/api/current/analytics/user_analytics/show.html.erb

```diff
@@ -17,7 +17,7 @@
 <% else %>
   <div class="card">
     <%= inline_svg('icon_new_bp.svg', classname: 'icon-float') %>
-    <h3>How many follow-up patients visited <%= @current_facility.name %>?</h3>
+    <h3><%= t('analytics.bps_recorded', facility_name: @current_facility.name) %></h3>
```

Comment:
analytics.followup_patients might be better here. Since we changed this graph to only show followups.